omero/developers/Web.txt

```diff
@@ -94,16 +108,16 @@ repository.
 
 There are 3 parts to each page: url, view and template. For example, this code
 below is for generating an HTML page of a Dataset (see screen-shot). If you
-have OMERO.web running, you can view the page under
-`http://servername.example.org/webtest/dataset/`\ `<datasetId>`.
+have OMERO.web running and webtest installed, you can view the page under
+`http://`\ `<servername>/webtest/dataset/`\ `<datasetId>`.
 
 -  **url** goes in omeroweb/omero_webtest/urls.py This maps the URL
    'webtest/dataset/<datasetId>/' to the View function
    'dataset', passing it the datasetId.
 
    ::
 
-       url( r'^dataset/(?P<datasetId>[0-9]+)/$', views.dataset ),
+       url( r'^dataset/(?P<dataset_id>[0-9]+)/$', views.dataset, name="webtest_dataset" ),
 
 -  **View** function, in omeroweb/omero_webtest/views.py. NB: @login\_required
```

Comment:
Why View and Template start with Capital and not Url
I will remove capital for view and template